boards/arm/s32k1xx/s32k146evb/src/s32k1xx_spi.c

```diff
@@ -146,13 +186,15 @@ uint8_t s32k1xx_lpspi0status(FAR struct spi_dev_s *dev, uint32_t devid)
 {
   return 0;
 }
-#endif
+#endif /* CONFIG_S32K1XX_LPSPI0 */
+
+/* LPSPI1 *******************************************************************/
 
 #ifdef CONFIG_S32K1XX_LPSPI1
-void s32k1xx_lpspi1select(FAR struct spi_dev_s *dev,
-                        uint32_t devid, bool selected)
+void s32k1xx_lpspi1select(FAR struct spi_dev_s *dev, uint32_t devid,
+                          bool selected)
 {
-  spiinfo("devid: %d CS: %s\n", (int)devid,
+  spiinfo("devid: %d, CS: %s\n", (int) devid,
```

Comment:
This and any sized types should use inttypes and the PRId32

Reply:
Thanks for reviewing, @davids5! I am not sure though what change you are requesting. Looks like many boards use the exact same code? Do you have any example of what this should look like instead?

Reply:
Have a look at https://github.com/apache/incubator-nuttx/blob/c55085c0d8864c9401de97872ab3463d474f262c/boards/arm/stm32l4/nucleo-l452re/src/stm32_adc.c#L233

It is used for sized types.

Reply:
I think I got it. Should be changed now.